### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by D300**
pull request worked after second commit - lets goooo

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -59,86 +59,96 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 1:
-
-//  action 2:
+//  Noun:       cocacola
+//  action 1:   cocacola sucks
+cocacola.suck();
+//  action 2:   cocacola overloads
+cocacola.overload();
+//  action 3:   cocacola sticks
+cocacola.stick();
```
Comment (matkatmusic) [inline main.cpp:77]: - [ ] camel-case this noun's name.

File: main.cpp [hunk 2]
```diff
@@ -59,86 +59,96 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 1:
-
-//  action 2:
+//  Noun:       bassdrum
+//  action 1:   bassdrum blasts
+bassdrum.blast();   
+//  action 2:   bassdrum gets muted (although its passive I think its ok)
+bassdrum.mute();
+//  action 3:   bassdrum hits
+bassdrum.hit();
```
Comment (matkatmusic) [inline main.cpp:113]: - [ ] camel-case this noun's name.

Comment (D300) [reply main.cpp]: camel-casing the left overs done

File: main.cpp (hunk 1 above)
Comment (D300) [reply main.cpp]: done too

**Pull request merged**